### PR TITLE
Add transfer hooks

### PR DESCRIPTION
**Pull request by nachoiacovino**
Updated the interface for Token so we can use the same one and it's the same one as the TS SDK

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -34,12 +34,12 @@ export type WalletAddress = string;
 export type ContractAddress = string;
 
 /**
- * The parameters to pass to the nft mint function.
+ * The parameters to pass to the nft mint and transfer functions.
  *
  * @beta
  */
-export type TokenMintParams = {
-  to: WalletAddress;
```
Comment (jnsdls) [inline src/types.ts:42]: does this not mean this becomes a breaking change? people would have to update their objects from `{to}` to `{toAddress}`?

I understand the change in itself but I don't think it's worth breaking inputs

**Review by jnsdls — CHANGES_REQUESTED** [review]
couple of small changes

File: src/hooks/async/token.ts [hunk 2]
```diff
@@ -189,3 +189,111 @@ export function useClaimToken<TContract extends TokenDrop>(
...
+ *     </button>
+ *   );
+ * };
+ * ```
+ *
+ * @param contract - an instance of a contract that extends the ERC20 spec (token, token drop, custom contract that follows the ERC20 spec)
+ * @returns a mutation object that can be used to mint a new NFT token to the connected wallet
+ * @beta
+ */
+export function useTransferBatchToken(contract: RequiredParam<Erc20>) {
```
[the remark this answers is not in the document]
Comment (nachoiacovino) [reply src/hooks/async/token.ts:277]: felt better to maintaint the use-functioName-token structure, but I can change it if you feel strongly about it

File: src/types.ts [hunk 3]
```diff
@@ -43,6 +43,16 @@ export type TokenMintParams = {
...
+/**
+ * The parameters to pass to the NFT transfer functions.
+ *
+ * @beta
+ */
+export type TokenParams = {
+  toAddress: WalletAddress;
+  amount: string | number;
+};
+
```
Comment (jnsdls) [inline src/types.ts:55]: let's merge this with tokenMintparams like you did but keep it `to` instead of `toAddress`

**Review by jnsdls — CHANGES_REQUESTED** [review]

**Review by jnsdls — APPROVED** [review]
LGTM

**Pull request merged**